src/current/v24.1/create-changefeed.md

```diff
@@ -148,72 +148,72 @@ Query parameters include:
+<a name="sasl-scopes">`sasl_scopes` | [Kafka]({% link {{ page.version.version }}/changefeed-sinks.md %}#kafka) | [`STRING`]({% link {{ page.version.version }}/string.md %}) | A list of scopes that the OAuth token should have access for. This parameter is only applicable with `sasl_mechanism=OAUTHBEARER`.
+<a name="sasl-token-url">`sasl_token_url` | [Kafka]({% link {{ page.version.version }}/changefeed-sinks.md %}#kafka) | [`STRING`]({% link {{ page.version.version }}/string.md %}) | Client token URL for OAuth authentication from a third-party provider. **Note:** You must [URL encode](https://www.urlencoder.org/) this value before passing in a URI. This parameter is only applicable with `sasl_mechanism=OAUTHBEARER`.
+<a name="sasl-user">`sasl_user`        | [Kafka]({% link {{ page.version.version }}/changefeed-sinks.md %}#kafka) | [`STRING`]({% link {{ page.version.version }}/string.md %}) | Your SASL username.
+<a name="sasl-password">`sasl_password`    | [Kafka]({% link {{ page.version.version }}/changefeed-sinks.md %}#kafka)                               | [`STRING`]({% link {{ page.version.version }}/string.md %})             | Your SASL password. **Note:** Passwords should be [URL encoded](https://wikipedia.org/wiki/Percent-encoding) since the value can contain characters that would cause authentication to fail.
+<a name="shared-access-key"><span class="version-tag">New in v24.1:</span>`shared_access_key` | [Azure Event Hubs]({% link {{ page.version.version }}/changefeed-sinks.md %}#azure-event-hubs) | [`STRING`]({% link {{ page.version.version }}/string.md %}) | The URL-encoded key for your Event Hub shared access policy.
+<a name="shared-access-key-name"><span class="version-tag">New in v24.1:</span>`shared_access_key_name` | [Azure Event Hubs]({% link {{ page.version.version }}/changefeed-sinks.md %}#azure-event-hubs) | [`STRING`]({% link {{ page.version.version }}/string.md %}) | The name of your Event Hub shared access policy.
+<a name="tls-enabled">`tls_enabled`      | [Kafka]({% link {{ page.version.version }}/changefeed-sinks.md %}#kafka), [Confluent Cloud]({% link {{ page.version.version }}/changefeed-sinks.md %}#confluent-cloud) | [`BOOL`]({% link {{ page.version.version }}/bool.md %})                 | If `true`, enable Transport Layer Security (TLS) on the connection to Kafka. This can be used with a `ca_cert` (see below).<br><br>For Confluent Cloud and Azure Event Hubs sinks, this is set to `true` by default.<br><br>**Default:** `false`
```

Comment:
Nerd comment: these opened A tags should ideally be closed eg ‘</a>’

Reply:
oops! Thanks for catching!